test/functional/interface_zmq_dash.py

```diff
@@ -226,6 +226,24 @@ def test_chainlock_publishers(self):
         assert_equal(uint256_to_string(zmq_chain_lock.blockHash), rpc_chain_lock_hash)
         assert_equal(zmq_chain_locked_block.hash, rpc_chain_lock_hash)
         assert_equal(zmq_chain_lock.sig.hex(), rpc_best_chain_lock_sig)
+
+        # Test getzmqnotifications RPC
```

Comment:
That's a bit weird place to put this test into imo... should be more like https://github.com/UdjinM6/dash/commit/d4fb7bbbc7b6abbdf27ae1636e92ef3df411d71f